src/main/resources/database/liquibase/changesets/user/changelog.xml

```diff
@@ -27,6 +27,13 @@
             <column name="modified_at" type="TIMESTAMP">
                 <constraints nullable="true"/>
             </column>
+            <column name="birth_date" type="DATE"/>
```

Comment:
This is incorrect modification. For each new change additional changeset should be created. Need to have a call.

Reply:
discussed it, leave it at that